src/frontends/pytorch/src/utils_quantize.cpp

```diff
@@ -237,12 +237,16 @@ std::shared_ptr<Node> u4_compression_stack(const OutputVector& list_elems, int64
             return nullptr;
     }
 
-    auto bitwise_shift = cast_fw_node(list_elems[1].get_node_shared_ptr(), "aten::bitwise_right_shift");
+    std::shared_ptr<Node> bitwise_shift =
```

Comment:
Do you really need to specify type? Isn't `auto` enough here?

Reply:
done